momento-sdk/src/intTest/java/momento/sdk/MomentoBatchUtilsIntegrationTest.java

```diff
@@ -160,14 +147,17 @@ void testBatchGetWithStringKeys_RequestsMoreThanMaxConn() {
             .build();
 
     // test data with more keys than max concurrent requests
+    Vector<String> keys = new Vector<>();
```

Comment:
```suggestion
    final List<String> keys = new ArrayList<>();
```
Is there a specific reason to use a Vector? I'm not sure I've ever seen one of those in use.

Reply:
That's what I used in my intro to Java class 22 years ago. I can use `ArrayList` lol

Reply:
Updated in [4eb3b71](https://github.com/momentohq/client-sdk-java/pull/379/commits/4eb3b719ac6a1e159d07a1e119369382a155b3f1)